source/renderer/app/api/wallets/requests/getWalletPublicKey.js

```diff
@@ -1,13 +1,18 @@
 // @flow
 import type { RequestConfig } from '../../common/types';
 import { request } from '../../utils/request';
+import type { GetWalletPublicKeyRequest } from '../types';
 
 export const getWalletPublicKey = (
   config: RequestConfig,
-  { walletId, role, index }: { walletId: string, role: string, index: string }
+  { walletId, index, passphrase, extended }: GetWalletPublicKeyRequest
 ): Promise<string> =>
-  request({
-    method: 'GET',
-    path: `/v2/wallets/${walletId}/keys/${role}/${index}`,
-    ...config,
-  });
+  request(
+    {
+      method: 'POST',
+      path: `/v2/wallets/${walletId}/keys/${index}`,
```

Comment:
@daniloprates @aleksandardjordjeviciohk this change has to be reverted. We need both this and the new Api methods as they do different things.

Reply:
We need both the "Create" and "Get" Api methods:
![Screenshot 2021-03-04 at 17 59 32](https://user-images.githubusercontent.com/376611/110000292-6a86c580-7d13-11eb-87e5-7ba9f83ddf99.png)

Reply:
Reverted old one and added new API method.